packages/scans/tsconfig.build.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "@tsconfig/node16/tsconfig.json",
+  "compilerOptions": {
+    "outDir": "dist"
+  },
+  "include": ["src/**/*"],
+  "exclude": ["src/**/*.test.ts"]
```

Comment:
Shouldn't the mocks being excluded as well?

Reply:
In any case not a blocker for me, can be tweaked after merge if needed.

Reply:
Good catch! Will address in next PR to not require an additional review of this one :)

Reply:
(And, just in case this isn't clear: This change wouldn't affect the users of the GitHub action, as the action is build via a bundler that anyways only bundles files that are used by the actual production code, so at that step all fixtures, tests, and test helpers would be ignored)